packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.gql.js

```diff
@@ -0,0 +1,62 @@
+import { gql } from '@apollo/client';
+
+export const SET_CUSTOMER_INFORMATION = gql`
+    mutation SetCustomerInformation(
+        $firstname: String!
+        $lastname: String!
+        $email: String!
+        $password: String!
+    ) {
+        updateCustomer(
+            input: {
+                firstname: $firstname
+                lastname: $lastname
+                email: $email
+                password: $password
```

Comment:
This is a little confusing; which of these mutations actually handles changing the password?

Reply:
This mutation to update customer informaton (firstname, lastname, email)

Reply:
This comment is fixed to a field named `password`, and there is a password input when you open the edit dialog that differs from the mocks. I see that the GraphQL mutation supports this field, but I'm not sure it actually does anything; if that is accurate it should be removed.

![Screen Shot 2020-09-17 at 8 49 10 AM](https://user-images.githubusercontent.com/462953/93480427-0d14ce80-f8c3-11ea-8b6d-fe96e77c79ac.png)

Reply:
I think @sirugh & @schensley are agree with that design.

Reply:
> This comment is fixed to a field named password, and there is a password input when you open the edit dialog that differs from the mocks. I see that the GraphQL mutation supports this field, but I'm not sure it actually does anything; if that is accurate it should be removed.

It was discovered that when attempting to change user email the password is required. To get around that backend requirement with little UI modification, and since this feature is long in development and has mocks expressing a different desire, we decided to just always render the password field. We only render the new password field when the "change password" button is clicked.